components/grid.css

```diff
@@ -0,0 +1,26 @@
+:root {
+    --grey-3: #333;
```

Comment:
variable naming could be done better. Try for eg: --color-grey-3

Reply:
will do better.